### PR TITLE
Add Quickstart tutorial

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

File: data/platforms.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- os: darwin
+  proc: amd64
+- os: linux
+  proc: amd64
+- os: linux
+  proc: 386
+- os: linux
+  proc: arm6
```
Comment (ColinSullivan1) [inline data/platforms.yaml]: The links generated by linux 386, and windows x86 didn't seem to work, but that could be the PR.

**Comment by lucperkins** [comment]
@ColinSullivan1 I'd say hold off for now, as it's still coming together. But that's a good catch on the download URLs! Fixed 😄

**Comment by lucperkins** [comment]
@ColinSullivan1 Okay, this is ready for review. It's fairly basic but could be used as a decent springboard for iteration.

File: content/documentation/quickstart.md [hunk 2]
```diff
@@ -53,26 +53,34 @@ This will run the NATS Server on port 4222 in debug mode. Debug mode isn't *nece
...
-### Receive messages synchronously
+> For more comprehensive instructions, see the [Receiving Messages](/documentation/writing_applications/subscribing/) guide.
 
-You can find instructions for your language of choice in the box below:
+### Receive messages synchronously {#sync}
 
 {{< partial "doc/subscribe_sync.html" >}}
 
-### Receive messages ascnchronously
+### Receive messages ascnchronously {#async}
```
Comment (gcolliso) [inline content/documentation/quickstart.md]: fix misspelling - asynchronously

File: content/documentation/quickstart.md [hunk 3]
```diff
@@ -53,26 +53,34 @@ This will run the NATS Server on port 4222 in debug mode. Debug mode isn't *nece
...
+
+{{< partial "doc/subscribe_async.html" >}}
 
-You can find instructions for your language of choice in the box below:
+## Next steps
 
-{{< partial "doc/subscribe_async.html" >}}
\ No newline at end of file
+Congrats! You've now successfully used NATS as a message broker using a client in your preferred language. But there's much more to explore in NATS.
```
Comment (gcolliso) [inline content/documentation/quickstart.md]: @ColinSullivan1 what might be a good next step? "Try blah blah blah..."

File: content/documentation/quickstart.md (hunk 2 above)
Comment (lucperkins) [reply content/documentation/quickstart.md]: Good catch

File: content/documentation/quickstart.md (hunk 3 above)
Comment (lucperkins) [reply content/documentation/quickstart.md]: What would you prefer to point users to?

**Pull request closed without merging**